pal/pacer/pacer-web.ts

```diff
@@ -81,37 +59,24 @@ export class Pacer {
 
     start (): void {
         if (this._isPlaying) return;
-        if (this._targetFrameRate === 60) {
-            const updateCallback = () => {
-                if (this._isPlaying) {
-                    this._rafHandle = this._rAF.call(window, updateCallback);
-                }
-                if (this._onTick) {
-                    this._onTick();
-                }
-            };
-            this._rafHandle = this._rAF.call(window, updateCallback);
-        } else {
-            const updateCallback = () => {
-                this._startTime = performance.now();
-                if (this._isPlaying) {
-                    this._stHandle = this._stTime(updateCallback);
-                }
-                if (this._onTick) {
-                    this._onTick();
-                }
-            };
+        const updateCallback = () => {
             this._startTime = performance.now();
-            this._stHandle = this._stTime(updateCallback);
-        }
+            if (this._isPlaying) {
+                this._stHandle = this._stTime(updateCallback);
+            }
+            if (this._onTick) {
+                this._onTick();
+            }
+        };
+        this._startTime = performance.now();
+        this._stHandle = this._stTime(updateCallback);
```

Comment:
I checked the function call here, each frame the call stack is
- _stTime
- setTimeoutRAF (generate handleRAF)
- handleRAF

So `handleRAF` is generated each frame, but it doesn't need to be.
Try to squeeze the process into one function call, I think you can implement everything in _stTime

Reply:
Then at least we shouldn't generate functions every frame, try to remove that logic, and make handleRAF consistent